test_data/generated/build/hello/hello.yaml

```diff
@@ -6,13 +6,13 @@ then:
   status: 200
   header:
   - name: date
-    value: Tue, 20 May 2025 17:29:55 GMT
+    value: Tue, 27 May 2025 11:41:48 GMT
```

Comment:
suggestion blocking: looks like we should bump all the other mk_data generated mocks?

Reply:
I am of the same two minds